database/exam/V1__exam_create_schema.sql

```diff
@@ -0,0 +1,582 @@
+/***********************************************************************************************************************
+  File: V1__exam_create_schema.sql
+
+  Desc: create the exam database schema
+
+***********************************************************************************************************************/
+-- Server version	5.6.27-log
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+CREATE SCHEMA IF NOT EXISTS exam
+  CHARACTER SET utf8
+  COLLATE utf8_unicode_ci;
+
+USE exam;
```

Comment:
So the more I think about it, the more I think this isn't the right repo for these scripts.  We can discuss further, but I think the right move is:
* Close this PR
* Move the `exam`, `exam_audit` and instructions for creating users to where the other db scripts are (which is `TDS_TestDeliverySystemDataAccess`, but that might not be right either)